source/help/settings/account-settings.rst

```diff
@@ -13,17 +13,17 @@ Settings to configure name, username, nickname, email and profile picture.
 Full Name
 ~~~~~~~~~
 
-Full names appear in the Direct Messages member list and team management modal. By default, you will receive mention notifications when someone types your first name. Entering a full name is optional.
+Full names appear in the Direct Messages member list and team management modal. By default, you will receive mention notifications when someone types your first name. Entering a full name is optional. Please note that `some unicode characters <https://www.w3.org/TR/unicode-xml/#Charlist>`_ are not supported. 
 
 Username
 ~~~~~~~~
 
-Usernames are unique identifiers appearing next to all posts. Pick something easy for teammates to recognize and recall. By default, you will receive mention notifications when someone types your username. In order to maintain message integrity, changing your username does not update @mentions in messages already posted. 
+Usernames are unique identifiers appearing next to all posts. Pick something easy for teammates to recognize and recall. By default, you will receive mention notifications when someone types your username. In order to maintain message integrity, changing your username does not update @mentions in messages already posted. Please note that `some unicode characters <https://www.w3.org/TR/unicode-xml/#Charlist>`_ are not supported. 
```

Comment:
Thanks @wiersgallak, wondering if there a source we can link that is more readable from the perspective of an end user given the target audience for this section of the docs? ie I think U+0340 is a backtick?

Reply:
This was a screenshot from QA testing on the PR so I believe it's already implemented in the product. So we can probably just add this note here in the docs to match

Reply:
@esethna Thanks for clarifying and great idea.  Please review the update I made in the username session.

Reply:
Looks good, thanks Katie!